data_collection/gazette/spiders/pr_campo_mourao.py

```diff
@@ -0,0 +1,48 @@
+from dateparser import parse
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCampoMouraoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4104303"
+    name = "pr_campo_mourao"
+    start_urls = ["https://campomourao.atende.net/?pg=diariooficial&pagina=1"]
+
+    def parse(self, response):
+        url_base = response.url[0:56]
+        page = int(response.url[56:])
+
+        list = response.xpath("//div[@class='nova_listagem ']/div[@class='linha']")
+
+        if not list:
+            return
+
+        for row in list:
+            date = row.xpath("//div[@class='info']/div[@class='data']/text()").get()
+            date = parse(date, languages=["pt"]).date()
+
+            edition_type = row.xpath(
+                "//div[@class='info']/div[@class='tipo']/text()"
+            ).get()
+            edition_number = row.xpath(
+                "//div[@class='info']/div[@class='titulo']/text()"
+            ).get()
+            edition_number = edition_number.split(" ")[1]
+
+            code = row.xpath("//button[@data-acao='download']/@data-codigo").get()
+            id = row.xpath("//button[@data-acao='download']/@data-id").get()
```

Comment:
Those xpaths are currently returning data from the first gazette in the page only. This is due to needing to work with relative xpaths. I suggest [this reading](https://docs.scrapy.org/en/latest/topics/selectors.html?highlight=relative#working-with-relative-xpaths) on Scrapy's documentation to understand why.

Also, simplifying your xpaths is usually better. As an example, if `.xpath(".//div[@class='data']/text()").get()` gets the date, you usually would prefer it instead of `.xpath(".//div[@class='info']/div[@class='data']/text()").get()`.

If you have further questions, ask away! :)

And a tip. To check if your spider is working correctly you could this command:
```shell
$ scrapy crawl pr_campo_mourao -o output.jsonlines -s LOG_FILE=logs.txt -s FILES_STORE=""
```

- `-o output.jsonlines`: Exports the output items to `data_collection/output.jsonlines` (useful to review if the items are being scraped as intended)
- `-s LOG_FILE=logs.txt`: Logs will be redirected to `data_collection/logs.txt` (useful to review if anything strange during the crawl occurred)
- `-s FILES_STORE=""`: Disables file downloading (useful to run the entire crawl and not fill your hard drive hehe)